src/main/java/com/pscheduler/server/model/GpaCourse.java

```diff
@@ -0,0 +1,42 @@
+package com.pscheduler.server.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Entity
+public class GpaCourse {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @NotNull
+    @Size(min = 2, max = 6)
+    private String subject;
+
+    @Version
+    private long version;
+
+    public GpaCourse() {
```

Comment:
Make this an empty constructor, we can test using hard coded data instead

Reply:
Done